ckanext/ontario_theme/templates/external/package/snippets/ontario_theme_access_level.html

```diff
@@ -7,12 +7,12 @@
     <div class="new-alert alert-{{pkg['access_level']}} alert-large alert-clear">
     {% if pkg['access_level'] == 'open' %}
       <strong>{{ _("Data Available") }}</strong>
-        <p>
-          {% trans license_url=pkg.license_url %}
-            The data described here is available for you to use. <a href="https://www.ontario.ca/page/ontarios-open-data-directive">[Learn more]</a><br />
-          <a href="{{ license_url }}">[See the licence for how you're allowed to use this data.]</a>
-          {% endtrans %}
-        </p>
+      <p>
+        {% trans license_url=_(pkg.license_url) %}
```

Comment:
Another option is to set the open-data-directive url as a variable `{% set open_data_url = _('https://www.ontario.ca/page/ontarios-open-data-directive') %}` at the top of this page and pass it through the href similar to `{{ license_url }}` so you wouldn't need to move translation or add more.

Reply:
Also, because this page is archived, we may need to change it to an updated url. It will be easier in the end to only have to change it twice rather than 6+ times.